sensu_plugin/utils.py

```diff
@@ -68,3 +68,63 @@ def deep_merge(dict_one, dict_two):
         else:
             merged[key] = value
     return merged
+
+
+def map_v2_event_into_v1(event):
+    '''
+    Helper method to convert Sensu 2.x event into Sensu 1.x event.
+    '''
+
+    # return the event if it has already been mapped
+    if "v2_event_mapped_into_v1" in event:
+        return event
+
+    # convert the event into a dict
+    event = json.loads(event)
+
+    # Trigger mapping code if enity exists and client does not
+    if "client" not in event and "entity" in event:
```

Comment:
Not sure how the python plugin does things in general for setting up handlers, but the skel ruby plugin will pre-populate the client attribute as an empty dict.  So in the ruby I specifically check if the client attribute is missing or if the client attribute value is empty.